src/ed25519/ecdsagen.c

```diff
@@ -54,7 +54,7 @@ static bool write_pem(FILE *fp, const char *type, void *vbuf, size_t size) {
 
 	while(size) {
 		size_t todo = size > 48 ? 48 : size;
-		b64encode(buf, base64, todo);
+		b64encode_tinc(buf, base64, todo);
```

Comment:
Yes. When I wrote this code I don't even think there was a standardized way to encode Ed25519 keys into PEM format. Maybe it's better to remove the PEM encoding for Ed25519 keys completely; by default `tinc init` generates it as a regular config variable. I think only `tinc fsck` uses this functionality, and that can be changed.